docs/source/get-started/package-managers.rst

```diff
@@ -3,20 +3,32 @@ Package Managers
+| Conda    | ``conda install conda-forge::kokkos`` | openMP (linux), Threads (windows), serial, CUDA | nope   | carterbox                             | `here <https://github.com/conda-forge/kokkos-feedstock/blob/main/recipe/build.sh>`_                      |
++----------+---------------------------------------+----------------------+--------+---------------------------------------+----------------------------------------------------------------------------------------------------------+
+| openSUSE | ``zypper install kokkos-devel``       | openMP, serial       | nope   | `mli@suse.com <mailto:mli@suse.com>`_ | `here <https://build.opensuse.org/projects/science/packages/kokkos/files/kokkos.spec?expand=1>`_         |
++----------+---------------------------------------+----------------------+--------+---------------------------------------+----------------------------------------------------------------------------------------------------------+
+| Gentoo   | ``emerge kokkos``                     | whatever enabled     | a bit  | tamiko                                | `here <https://gitweb.gentoo.org/repo/gentoo.git/tree/dev-cpp/kokkos/kokkos-4.3.1.ebuild>`_              |
++----------+---------------------------------------+----------------------+--------+---------------------------------------+----------------------------------------------------------------------------------------------------------+
+| Arch     | ``pacman -S kokkos``                  | Threads, serial      | maybe  | carlosal1015                          | `here <https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=kokkos>`_                                  |
++----------+---------------------------------------+----------------------+--------+---------------------------------------+----------------------------------------------------------------------------------------------------------+
+| Mac Port | ``port install kokkos-devel``         | openMP, serial       | no     | MarcusCalhoun-Lopez                   | `here <https://github.com/macports/macports-ports/blob/master/devel/kokkos/Portfile>`_                   |
++----------+---------------------------------------+----------------------+--------+---------------------------------------+----------------------------------------------------------------------------------------------------------+
+| Spack    | ``spack install kokkos``              | whatever enabled     | yes    | cedricchevalier19, nmm0, lucbv        | `here <https://github.com/spack/spack/blob/develop/var/spack/repos/builtin/packages/kokkos/package.py>`_ |
++----------+---------------------------------------+----------------------+--------+---------------------------------------+----------------------------------------------------------------------------------------------------------+
```

Comment:
I am not sure that we want to put this kind of table in the primary documentation:
 - Some information is targeted to end-users (what packages exist and if they are "vetted" or not)
 - Other information related to collaboration and contributions: maintainers, source link.

For the user, I would rather have a sentence explaining that `Kokkos` might be available in distribution packages. Still, we do not know their usability deeply for now, so we advise building from source.

However, having a page for packagers makes sense, but I am not sure that we want to duplicate a table that is mostly dynamically available on [repology](https://repology.org/project/kokkos/versions). Here, the "vetted" status might be more apparent. Currently, it is not obvious what "a bit" vs "maybe" means.